Source/Examples/WPF.SharpDX/InstancingDemo/MainWindow.xaml

```diff
@@ -7,7 +7,7 @@
         mc:Ignorable="d"         
         Title="{Binding Title}" Height="500" Width="800">
     <Window.Resources>
-        <x:StaticExtension Member="hx:Techniques.RenderTechniques" x:Key="RenderTechniques"/>

```

Comment:
Because we no longer have a static `Techniques` class, we use our `TechniquesManager` instance. We use a converter to converter a `KeyValuePair` object representing the items in the `ComboBox` back to a `RenderTechnique`.